services/user.py

```diff
@@ -0,0 +1,49 @@
+from db.models import User
+from django.contrib.auth import get_user_model
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None
+) -> None:

```

Comment:
please try to use one codestyle) 
![image](https://github.com/mate-academy/py-tickets-and-orders/assets/80958128/1204e046-3f70-4403-92d3-913752cbee2a)
or next
![image](https://github.com/mate-academy/py-tickets-and-orders/assets/80958128/428154da-7699-461c-a422-4c5bb42f43f0)